src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -2129,6 +2129,10 @@ def load_arguments(self, _):
             options_list=["--node-vm-size", "-s"],
             completer=get_vm_size_completion_list,
         )
+        c.argument(
+            "gpu_driver",
```

Comment:
I believe you also need to declare this parameter in the entry function corresponding to the command, which is `aks_agentpool_update` in custom.py